src/app/_components/FrontLayout/FrontLayout.tsx

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import { Grid, Typography, Box, Divider, useMediaQuery } from "@mui/material";
+import Logo from "~/assets/images/logo_font.svg";
+import Image from "next/image";
+import theme from "~/theme";
+
+const FrontLayout = (): JSX.Element => {
+  const mobileSmallScreen = useMediaQuery(theme.breakpoints.down("sm"));
+
+const MobileFooterContainer = () => {
+  return (
+    <Box id="have-an-account" sx={{
+      textAlign: "center",
+      marginTop: "auto",
+      marginBottom: "1rem",
+      position: "absolute",
+      bottom: "1rem",
+      width: mobileSmallScreen ? "100vw" : "100%",
+      borderTop: "1px solid #CCC",
+      paddingTop: "13px",
+      height: mobileSmallScreen ? "10px" : "40px",
+    }}
+    > Have an account? Container
+    </Box>
+  )
+}
+
+  return (
+    <>
+      <Grid
+        container
+        justifyContent="center"
+        alignItems="center"
+        style={{ minHeight: "100vh" }}
+      >
+        <Box
+          id="borderline"
+          sx={{
+            border: mobileSmallScreen ? "none" : "1px solid #CCC",
+            maxHeight: "812px",
+            height: "90vh",
+            margin: "auto",
+            display: "flex",
+            flexDirection: "column",
+            justifyContent: "center",
+            alignItems: "center",
+            padding: "1rem",
+            position: "relative",
+            "@media (min-width: 375px)": {
+              width: "375px",
+            },
+          }}
+        >
+          <Box
+            id="logo-container"
+            mb={6}
```

Comment:
might be a bit too much margin at the bottom

Reply:
According design, it's 51px in between logo and the text. I think it could be enough because below logo, there are only two lines of texts and one single button element will be added.